plugin.xml

```diff
@@ -22,17 +22,36 @@
         </config-file>
 
         <framework src="com.github.shogo4405.HaishinKit~kt:haishinkit:0.10.4" />
-        <framework src="src/android/build-extras.gradle" custom="true" type="gradleReference" />
+      <!--  <framework src="src/android/build-extras.gradle" custom="true" type="gradleReference" />-->
```

Comment:
I would prefer removing the code instead of commenting - but let me know if there's a good reason

Reply:
@ediboy the problem regarding this, was because this instruction is on branch [create-cordova-package] but the file is only added on branch implement-android-native-v2. 
So on this branch we dont have this file, which when adding the plugin to cordova project fails due not finding the file hence I have commented this line.

When we merge, we shall see this conflict and addressed by then, I hope it makes sense